src/config/tests.rs

```diff
@@ -56,12 +56,12 @@ fn test_modmap_multi_purpose_key() {
     modmap:
       - remap:
           Space:
-            held: Shift_L
-            alone: Space
+            held: [Shift_L]
+            alone: [Space]
       - remap:
           Muhenkan:
-            held: Alt_L
-            alone: Muhenkan
+            held: [Alt_L]
+            alone: [Muhenkan]
```

Comment:
Can you also leave a test that parses the non-vector case?

Reply:
done

Reply:
I meant a test case without `[` and `]`. Could you just add a new test instead of rewriting an existing test?

Reply:
done

Reply:
Thanks!